custom/icds_reports/views.py

```diff
@@ -742,6 +743,9 @@ def post(self, request, *args, **kwargs):
         if indicator == AWW_INCENTIVE_REPORT:
             if not sql_location or sql_location.location_type_name != LocationTypes.BLOCK:
                 return HttpResponseBadRequest()
+            today = datetime.now(timezone('Asia/Kolkata'))
```

Comment:
Thoughts on standardizing this:

https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/const.py#L10
https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/__init__.py#L67
https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/__init__.py#L778

https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/templatetags/icds_format.py#L102-L103